pkg/cmd/pr/shared/commentable.go

```diff
@@ -61,15 +67,15 @@ func CommentablePreRun(cmd *cobra.Command, opts *CommentableOptions) error {
 
 	if inputFlags == 0 {
 		if !opts.IO.CanPrompt() {
-			return &cmdutil.FlagError{Err: errors.New("--body or --web required when not running interactively")}
+			return &cmdutil.FlagError{Err: errors.New("--body, --body-file or --web required when not running interactively")}
 		}
 		opts.Interactive = true
 	} else if inputFlags == 1 {
 		if !opts.IO.CanPrompt() && opts.InputType == InputTypeEditor {
-			return &cmdutil.FlagError{Err: errors.New("--body or --web required when not running interactively")}
+			return &cmdutil.FlagError{Err: errors.New("--body, --body-file or --web required when not running interactively")}
 		}
 	} else if inputFlags > 1 {
-		return &cmdutil.FlagError{Err: fmt.Errorf("specify only one of --body, --editor, or --web")}
+		return &cmdutil.FlagError{Err: fmt.Errorf("specify only one of --body, --body-file, --editor, or --web")}
```

Comment:
Nit: could you add backticks around flag expressions in these error messages? We've recently started using backticks more in docs and error messages to denote code or user input, e.g. `` to enable X, please use the `--enable-X` flag ``, but we haven't updated all of the old messages yet